tests/h/streamer/messages_speed_test.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from _datetime import datetime
+from pyramid import security
+
+from h.streamer.messages import handle_annotation_event
+from h.streamer.websocket import WebSocket
+from h.websocket import create_app
+from tests.common.fixtures.elasticsearch import ELASTICSEARCH_INDEX, ELASTICSEARCH_URL
+
+
+@pytest.mark.skip("Only of use during development")
+class TestHandleAnnotationEventSpeed:  # pragma: no cover
+    @pytest.mark.parametrize("reps", (1, 16, 256, 4096))
+    @pytest.mark.parametrize("action", ("create", "delete"))
+    def test_speed(self, db_session, registry, socket, message, action, reps):
+        sockets = list(socket for _ in range(reps))
+        message["action"] = action
+
+        start = datetime.utcnow()
+        handle_annotation_event(message, sockets, registry, db_session)
+        diff = datetime.utcnow() - start
+
+        assert socket.send_json.count == reps
+
+        millis = diff.seconds * 1000 + diff.microseconds / 1000
+        print(
+            f"{action} x {reps}: {millis} ms, {millis/reps} ms/item, {reps/millis*1000} items/sec"
+        )
```

Comment:
Speed tests are an interesting idea. It's a shame that, since they won't be run on CI, there's nothing to protect them from being broken the next time someone tries to run them

Reply:
Yeah, they are kind of pain because what's fast enough? All the test fixturing and tooling is just a really convenient way of writing this stuff.

I mean we _could_ run it?